resources/views/partials/top_nav.blade.php

```diff
@@ -221,15 +224,21 @@
         @if (config('donation.is_enabled'))
             <li class="top-nav__dropdown">
                 @php
-                    $sum = App\Models\Donation::whereMonth('created_at', date('m'))
-                        ->whereYear('created_at', date('Y'))
-                        ->where('status', App\Models\Donation::APPROVED)
-                        ->with('package')
-                        ->get()
-                        ->sum(function ($donation) {
-                            return $donation->package->cost;
-                        });
-                    $percentage = $sum ? min(100, number_format(($sum / config('donation.monthly_goal')) * 100)) : 0;
+                    $goal = config('donation.goal');
+
+                    $startDate = config('donation.start_date');
+
+                    $sum = Donation::join('donation_packages', 'donations.package_id', '=', 'donation_packages.id')
+                        ->where('donations.status', Donation::APPROVED)
+                        ->when(
+                            config('donation.goal_type') === 'yearly',
```

Comment:
Yearly can probably be renamed to "one-time" or "drive", as it's not actually yearly.